core/src/main/java/greencity/controller/SuperAdminController.java

```diff
@@ -145,9 +155,10 @@ public ResponseEntity<HttpStatus> deleteTariffService(
     @PreAuthorize("@preAuthorizer.hasAuthority('EDIT_PRICING_CARD', authentication)")
     @PutMapping("/editTariffService/{id}")
     public ResponseEntity<GetTariffServiceDto> editTariffService(
-        @RequestBody EditTariffServiceDto editTariff, @Valid @PathVariable Integer id,
+        @RequestBody TariffServiceDto dto,
+        @Valid @PathVariable Integer id,
```

Comment:
I see you use [Integer](https://github.com/ita-social-projects/GreenCityUBS/pull/990/files#diff-ea3792e71d2d7d18600958e51eae2e3f0316c18162b9d7e4ee58c871139b7058R159) and [long](https://github.com/ita-social-projects/GreenCityUBS/pull/990/files#diff-ea3792e71d2d7d18600958e51eae2e3f0316c18162b9d7e4ee58c871139b7058R110) for id. Why?

Reply:
id has an integer type for Bag and a long type for TariffsInfo